src/components/HistoryCard/HistoryCard.jsx

```diff
@@ -0,0 +1,33 @@
+import { useHistory } from '../../context';
+import { thumbnailLink } from '../../utils/video-details-services';
+import './HistoryCard.css';
+
+const HistoryCard = ({ video }) => {
+    const { _id, title, description, creator } = video;
+
+    const reduceTitleLength = (title) => {
+        if (title.length > 50) return title.substring(0, 47) + '...';
+        return title;
+    };
```

Comment:
You can use CSS ellipsis here to truncate the title.

Reply:
I have kept it this way so that I get to control the number of characters. Later on i'll extract out to a util function.